Combinatorial_map/include/CGAL/Cell_iterators.h

```diff
@@ -86,7 +86,8 @@ namespace CGAL {
                               Tag_true>::value);
       CGAL_assertion(amap.is_whole_map_unmarked(mcell_mark_number));
 
-      mark_cell<Map,i,dim>(amap, adart, mcell_mark_number);
+      if(this->cont())
+      { mark_cell<Map,i,dim>(amap, adart, mcell_mark_number); }
```

Comment:
Wouldn't it be better to have the test inside the function (as this is anyway provided) and this will in one go protect all calls to the function.

Reply:
the mark_cell function takes a descriptor as input; and has a requirement that this descriptor is "valid".
thus I think the current use is correct.